primo/data_parser/well_data.py

```diff
@@ -915,6 +924,92 @@ def compute_priority_scores(self, impact_metrics: ImpactMetrics):
                 )
                 raise_exception(msg, ValueError)
 
+    def _process_dac_data(self):
+        """
+        processes the dac data
+        """
+        self._append_fed_dac_data()
+        return
+
+    def _compute_well_count_score(self, metric: Metric, metrics: ImpactMetrics):
```

Comment:
I think we can remove arguments from this method now. `metrics` is now available in `config`

Reply:
Addressed (when I push)...